src/components/visualizations/chart/events/load.ts

```diff
@@ -0,0 +1,24 @@
+// (C) 2007-2018 GoodData Corporation
+import { Chart } from 'highcharts';
+import get = require('lodash/get');
+
+// TODO: once Highcharts is upgraded to 7.0.1, 'yAxis.className' can be used to avoid registering 'load' event
+// We can add 'className' to 'yAxis' in 'chartOptionsBuilder.ts' and 'customConfiguration.ts'
+// (https://api.highcharts.com/highcharts/yAxis.className is supported since 5.0.0)
+/**
+ * Add class to Y axis to support selenium test for dual axes chart
+ * @param chart
+ */
+function addClassToYAxis(chart: Chart): void {
+    const yAxis: any[] = get(chart, 'yAxis', []);
+
+    yAxis.forEach((axis: any) => {
+        const axisType: string = axis.opposite ? 'secondary' : 'primary';
+        axis.axisGroup.addClass(`s-highcharts-${axisType}-yaxis`);
+        axis.labelGroup.addClass(`s-highcharts-${axisType}-yaxis-labels`);
+    });
+}
+
+export default function handleChartLoad() {
```

Comment:
We should be able to unit test this exported function.

Reply:
Could you explain more what unit test cases we need here?

Reply:
@strouhaljan 
Unfortunately, I cannot do as your guide.
Highcharts does not fire `load` event in unit test with `mount` or `render` of enzyme, so that I cannot spy or mock the method to run `expect`.
Do you know any existing cases in our code base? so I can refer to it.

Meanwhile, I add some cases to `highChartsCreators.spec.ts` to assert event registered for supported chart.

Reply:
This is so neat.
Thank you :)